src/esp/gfx_batch/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+set(
+  gfx_batch_SOURCES
+  Renderer.cpp
+  Renderer.h
+  RendererStandalone.cpp
+  RendererStandalone.h)
+
+add_library(
+  gfx_batch STATIC
+  ${gfx_batch_SOURCES}
+)
+# For esp/core/configure.h
+# TODO move configure.h directly to esp/
+target_include_directories(gfx_batch PUBLIC ${PROJECT_BINARY_DIR})
+if(BUILD_WITH_CUDA)
+  target_include_directories(
+    gfx_batch PRIVATE ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}
+                      ${CMAKE_CURRENT_LIST_DIR}/cuda_helpers
+  )
+
+  target_link_libraries(gfx_batch PUBLIC ${CUDART_LIBRARY})
+endif()
+
+target_link_libraries(
```

Comment:
Is this where the naive Habitat programmer would add `core` and other stuff? Can we add a scary comment here to discourage this?

Reply:
(Note added.)